### PR TITLE
2286 dq indicate mapped fields

### PR DESCRIPTION
**Pull request by adrian-lara**
#### Any background context you want to provide?
When defining DQ Rules, we realized it would be helpful to flag fields that were not mapped by an organization. 

See attached issue for more details.

#### What's this PR do?
This PR introduces an indicator on the DQ admin page that shows which fields have been mapped to during file imports. In the backend, the mapped, or "used", columns are flagged by checking if `ColumnMapping.objects.filter(column_mapped=[col_object]).exists()`. [This check existed already and didn't need to be written.](https://github.com/SEED-platform/seed/blob/4baeb3441f92a4e886d2d28c515dde39289f05c7/seed/models/columns.py#L1321-L1323)

Note that no checks are performed on the actual inventory records as that operation could potentially be very expensive.

#### How should this be manually tested?
Visit the DQ admin page, and see that a ** is added to indicate fields that have not been mapped. Spot check to confirm fields are correctly flagged.

#### What are the relevant tickets?
#2286 

#### Screenshots (if appropriate)
Screen Shot 2020-07-15 at 1 21 30 PM

### CODE REVIEW

File: seed/views/properties.py [hunk 1]
```diff
@@ -814,7 +816,7 @@ def columns(self, request):
               paramType: query
         """
         organization_id = int(request.query_params.get('organization_id'))
-        only_used = request.query_params.get('only_used', False)
+        only_used = json.loads(request.query_params.get('only_used', 'false'))
```
Comment (adrian-lara) [inline seed/views/properties.py:819]: Before, sending `"false"` to this endpoint actually ended up making `only_used` be a truthy, non-empty string. Note, this fix happened for the corresponding endpoint in v3, so no changes needed there.

File: seed/static/seed/js/seed.js [hunk 2]
```diff
@@ -941,6 +941,26 @@ SEED_app.config(['stateHelperProvider', '$urlRouterProvider', '$locationProvider
               });
             }
           }],
+          used_columns: ['$stateParams', 'inventory_service', function ($stateParams, inventory_service) {
```
Comment (adrian-lara) [inline seed/static/seed/js/seed.js:944]: `used_columns` is the same as `columns` just above this except the `get_[inv_type]_columns_for_org` methods take a `true` arg and sorting isn't performed.

**Comment by axelstudios** [comment]
Updated screenshot with grouping:
image

**Comment by RDmitchell** [comment]
@axelstudios / @adrian-lara -- wow, this looks great !! Thank you !!

**Comment by RDmitchell** [comment]
Instance: dev1
SHA: fd75dea4
Org: LBNL Test 200

This is working as described in the PR, ie, grouped by Mapped and Not Mapped. I think this is a decent way to group the fields, but when there are a lot of fields that are mapped, it's hard to see which category a field that you are looking for belongs to.

But it's fine for now. 

image

**Pull request merged**